src/test/java/marquez/core/services/JobServiceIntegrationTest.java

```diff
@@ -66,7 +66,8 @@ public void tearDown() {
   @Test
   public void testCreate() throws UnexpectedException {
     Job job = Generator.genJob(namespaceID);
-    jobService.createJob(namespaceName, job);
+    Job jobReturned = jobService.createJob(namespaceName, job);
+    assertNotNull(jobReturned.getCreatedAt());
     Optional<Job> jobFound = jobService.getJob(namespaceName, job.getName());
     assertTrue(jobFound.isPresent());
     assertEquals(job.getName(), jobFound.get().getName());
```

Comment:
Would also be cool to test if the result of a create operation is equal to the result of a get operation. I can't think of a reason why they wouldn't be equal.

Reply:
Makes sense, we can add this